runtime/common/Cargo.toml

```diff
@@ -16,12 +16,12 @@ codec = {package = "parity-scale-codec", version = "3.0.0", default-features = f
 scale-info = {version = "2.5.0", default-features = false, features = ["derive"]}
```

Comment:
Minor update to `scale-info`.

Reply:
Fixed in [1c74d00](https://github.com/paritytech/extended-parachain-template/pull/54/commits/1c74d008b375d31f5677f61f93e0986f9a817b20).